lib/Varien/Data/Form/Element/Multiselect.php

```diff
@@ -57,8 +57,10 @@ public function getElementHtml()
     {
         $this->addClass('select multiselect');
         $html = '';
-        if ($this->getCanBeEmpty() && empty($this->_data['disabled'])) {
-            $html .= '<input type="hidden" name="' . parent::getName() . '" value="" />';
+        if ($this->getCanBeEmpty()) {
+            $html .= '<input type="hidden" name="' . parent::getName() . '" value=""';
+            $html .= (!empty($this->_data['disabled'])) ? ' disabled="disabled"' : '';
```

Comment:
Is this simpler and more readable?
```suggestion
            $html .= empty($this->_data['disabled']) ? '' : ' disabled="disabled"';
```